spark/src/main/scala/org/allenai/pipeline/spark/CreateRddArtifacts.scala

```diff
@@ -0,0 +1,45 @@
+package org.allenai.pipeline.spark
+
+import org.allenai.pipeline._
+import org.allenai.pipeline.s3.{ S3Credentials, S3Config }
+
+import scala.reflect.ClassTag
+
+import java.io.File
+import java.net.URI
+
+/** Created by rodneykinney on 5/24/15.
+  */
+object CreateRddArtifacts {
```

Comment:
Similarly, support for creating Artifacts that support persistence of Spark RDDs are enabled by mixing in the `SparkPipeline` trait from the spark sub-project.